docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4728,7 +4729,7 @@ defined as follows:
 ~ Begin Proto
 message WriteRequest {
   uint64 device_id = 1;
-  uint64 role_id = 2;
+  string role = 6;
```

Comment:
I am fine with the current solution of not including deprecated fields here. it is consistent with the fact that we do not describe deprecated fields such as `role_id` in the spec.